src/map/skill.h

```diff
@@ -2154,8 +2155,7 @@ struct skill_interface {
 	void (*attack_combo2_unknown) (int *attack_type, struct block_list* src, struct block_list *dsrc, struct block_list *bl, uint16 *skill_id, uint16 *skill_lv, int64 *tick, int *flag, int *combo);
 	void (*attack_display_unknown) (int *attack_type, struct block_list* src, struct block_list *dsrc, struct block_list *bl, uint16 *skill_id, uint16 *skill_lv, int64 *tick, int *flag, int *type, struct Damage *dmg, int64 *damage);
 	int (*attack_copy_unknown) (int *attack_type, struct block_list* src, struct block_list *dsrc, struct block_list *bl, uint16 *skill_id, uint16 *skill_lv, int64 *tick, int *flag);
-	int (*attack_dir_unknown) (int *attack_type, struct block_list* src, struct block_list *dsrc, struct block_list *bl, uint16 *skill_id, uint16 *skill_lv, int64 *tick, int *flag);
```

Comment:
Found out this might be used by some plugins, will instead add proper documentation that mentions why it exists, so people like me don't accidentaly remove it.
(Correct me if I'm wrong or give a 👍  emote)

Reply:
Yes, all unknown functions are meant for plugins and shall be kept.

Reply:
Undoing.

Reply:
See 2d3b9ac4e5f851a6c956ecd52a9c9b87df0ca108